Cargo.toml

```diff
@@ -140,11 +166,40 @@ name = "client_json"
 path = "examples/client_json.rs"
 required-features = ["full"]
 
+[[example]]
+name = "client_json_tracing_json"
+path = "examples/client_json_tracing_json.rs"
+required-features = ["full",
+    "tracing/max_level_trace",
+    ]
+
+[[example]]
+name = "client_json_tracing_off"
+path = "examples/client_json_tracing_off.rs"
+required-features = ["full",
+    "tracing/max_level_off",
+    "log/max_level_off",
+    ]
+
+[[example]]
+name = "client_json_tracing_otel"
+path = "examples/client_json_tracing_otel.rs"
+required-features = ["full",
+    "tracing/max_level_trace",
+    ]
+
```

Comment:
take it or leave it: i think it might be worth putting all the `tracing` examples in a `examples/tracing` directory, so it's clear they're all related to various `tracing`-related configurations?

Reply:
> take it or leave it: i think it might be worth putting all the `tracing` examples in a `examples/tracing` directory, so it's clear they're all related to various `tracing`-related configurations?

OT but I'll take it into account closer to the time a final PR is ready.